workflows/Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack/workflow.spec.yaml

```
@@ -0,0 +1,26 @@
+extension: workflow
+products: ["insightconnect"]
+name: Blacklist_Indicators_with_Microsoft_Defender_ATP_from_Slack
+title: Blacklist Indicators with Microsoft Defender ATP from Slack
+description: Use the simplicity of Slack to blacklist and unblacklist indicators such as SHA1 hashes, SHA256 hashes, domains, URLS and IP addresses with Microsoft Defender ATP.
```

```suggestion
description: Use the simplicity of Slack to blacklist and unblacklist indicators such as hashes, domains, URLs, and IP addresses with Microsoft Defender ATP.
```
Just a bit more succinct so it fits better